src/System.Private.ServiceModel/src/System/ServiceModel/Channels/Connection.cs

```diff
@@ -614,10 +614,10 @@ public void EndWrite()
             }
         }
 
-        private void OnWrite(Task antecedant, Object state)
+        private void OnWrite(Task antecedent, Object state)
         {
             Contract.Requires(_writeResult == null, "StreamConnection: OnWrite called twice.");
-            _writeResult = antecedant;
+            _writeResult = antecedent;
```

Comment:
Two more instances of this typo on lines 724 and 727, can you fix those as well?